calicoctl/calico_ctl/checksystem.py

```diff
@@ -246,3 +253,37 @@ def _check_etcd_version():
             system_ok = False
 
     return system_ok
+
+
+def _check_kernel_version():
+    """
+    Check that the kernel version meets the minimum required version for Calico.
+    Valid kernel versions have 1 Kernel digit, 1-2 Major digits, and optionally
```

Comment:
Is it right to talk about the number of digits?  Surely the key thing is that each segment is a number, not a set number of digits?